src/plugins/intel_npu/src/plugin/npuw/serialization.cpp

```diff
@@ -0,0 +1,184 @@
+// Copyright (C) 2024-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "serialization.hpp"
+
+#include "intel_npu/config/config.hpp"
+#include "logging.hpp"
+#include "openvino/op/constant.hpp"
+#include "spatial.hpp"
+
+void ov::npuw::s11n::write(std::ostream& stream, const std::streampos& var) {
+    stream.write(reinterpret_cast<const char*>(&var), sizeof var);
+}
```

Comment:
what is streampos, why don't you just put std::uint32_t instead.

Reply:
On some platforms it can be bigger than uint32_t